galleries/plot_types/basic/scatter_plot.py

```diff
@@ -11,12 +11,14 @@
 plt.style.use('_mpl-gallery')
 
 # make the data
+rng = np.random.default_rng(seed=None)
 np.random.seed(3)
```

Comment:
If we go through with the change, this should probably be deleted:

```suggestion
```

(which, contrary to our docs and @tacaswell's suggestion on the line above, doesn't actually use 19680801, though perhaps should have.)

Reply:
yes,I forget.